app/services/clients/ledger.client.test.js

```diff
@@ -62,13 +62,10 @@ class MockClient {
   async get (url, description) {
     let dataResponse
 
-    if (url.match(/\.*\/event/)) { dataResponse = ledgerTransactionEventsFixture }
-    else if (url.match(/\/transaction\//)) dataResponse = { validCreatedTransactionDetailsResponse }
-    else if (url.match(/\/report\/transactions-summary/)) { dataResponse = validTransactionSummaryResponse }
-    else if (url.match(/\/agreement/)) dataResponse = { validTransactionSummaryResponse }
+    if (url.match(/\.*\/event/)) { dataResponse = ledgerTransactionEventsFixture } else if (url.match(/\/transaction\//)) dataResponse = { validCreatedTransactionDetailsResponse }
```

Comment:
shouldn't these else ifs each be on a new line?

Reply:
This is one of those unrelated files changed by running `npm run lint`.... but yeah I can change it.

Reply:
wow I've always been told to put that sort of thing on separate lines, would never have expected a linter to make that kind of change. Something to watch out for I guess.

Reply:
Fixed